src/config.rs

```diff
@@ -125,6 +128,7 @@ fn parse_format(input: &str) -> Result<Format, String> {
         "libtest" => Ok(Format::LibTest),
         "json" => Ok(Format::Json),
         "tap" => Ok(Format::Tap),
+        "compact" => Ok(Format::Compact),
```

Comment:
Do we also need to update the help string?

Reply:
Good catch, thanks. Will do